README.md

```diff
@@ -0,0 +1,54 @@
+# Dangl.XRechnung.Examples
+
+> An online version of the converter is freely accessible at [my blog](https://blog.dangl.me/xrechnung-converter/)  TODO: ADD A WORKING LINK
```

Comment:
Just use https://web-gaeb.de/, we've got an XRechnung viewer already there. It's just in the beta version yet, but once the UI PRs are done we'll release it, probably even this week.